contracts/event/MultipleResultsEvent.sol

```diff
@@ -543,6 +503,11 @@ contract MultipleResultsEvent is NRC223Receiver, Ownable {
             nextThreshold,
             arbitrationEndTime);
 
+        // Refund difference over threshold
+        if (diff > 0) {
+            INRC223(_bodhiTokenAddress).transfer(from, diff);
```

Comment:
This is buggy, you transfer back the over paid, but the `_currentVotingRoundTotals[resultIndex] >
            _eventRounds[_currentRound].consensusThreshold`, we need to check in `vote()` function

Reply:
it is checked in the vote func L460. voteSetResult() is not called if its not voted over the threshold.

Reply:
fixed, good catch